### PR TITLE
Gift Entry: add tokenize widget stub

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes
- Add the tokenize card widget stub so it can be included in a template and form
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by psadie-sfdo** [comment]
**lurch: attach W-039739

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HsmKQAS" target="_blank">W-039739</a>

**Comment by ptewson-sfdo** [comment]
@melissabarber This is our first payments story, but includes a custom label. Including you early and often!

File: src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js-meta.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>47.0</apiVersion>
```
Comment (ptewson-sfdo) [inline src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js-meta.xml]: Maybe we bump to v48?

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -5650,6 +5714,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Label for selecting fields to include as columns</shortDescription>
         <value>Select Fields to Display</value>
     </labels>
+    <labels>
+        <fullName>geHeaderPaymentServices</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>iFrame header for Payment Services information</shortDescription>
+        <value>Powered by Payment Services</value>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels]:
```suggestion
        <value>Powered by {0}</value>
```

Comment (melissabarber) [reply src/labels/CustomLabels.labels]: @psadie-sfdo I think we were going to do a variable here and pull in the common label for Payment Services. That way, if (when) Payment Services changes to a different name, we can just update the common label and have this one match.

File: src/labels/CustomLabels.labels [hunk 3]
```diff
@@ -5889,6 +5961,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Help text for Form Fields Field Label</shortDescription>
         <value>This shows the Object Group and target field API Name. You can edit the user-facing label below.</value>
     </labels>
+    <labels>
+        <fullName>geHelpTextPaymentServices</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Temporary note about only being able to use Payment Services with single gifts</shortDescription>
+        <value>Note: This field bundle is only available for single gifts and will not show up when entering batches gifts.</value>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels]:
```suggestion
        <value>Note: This field bundle is only available for single gifts and will not show up when entering batches of gifts.</value>
```

**Pull request merged**